README.yaml

```diff
@@ -51,7 +51,9 @@ description: |-
 usage: |-
   ```hcl
   module "dynamodb_autoscaler" {
-    source                       = "git::https://github.com/cloudposse/terraform-aws-dynamodb-autoscaler.git?ref=master"
+    source = "cloudposse/dynamodb-autoscaler/aws"
```

Comment:
Our regex should capture the whitespace to preserve the formatting

Reply:
With the comment below it, the example is formatted as `terraform fmt` would format it.

Reply:
Technically correct ;)